src/js/components/tasks/detail.tsx

```diff
@@ -812,7 +822,7 @@ const TaskDetail = (
 
   return (
     <DocumentTitle
-      title={` ${task.name} | ${epic.name} | ${project.name} | ${i18n.t(
+      title={` ${task.name} | ${epic?.name} | ${project.name} | ${i18n.t(
         'Metecho',
       )}`}
```

Comment:
I think we probably don't want `undefined` in the page title. Maybe something like:
```suggestion
      title={
        epic
          ? `${task.name} | ${epic.name} | ${project.name} | ${i18n.t(
              'Metecho',
            )}`
          : `${task.name} | ${project.name} | ${i18n.t('Metecho')}`
      }
```